Volha_Ivanova/Module_1_Java/Java_Fundamentals/HomeWork1/src/com/epam/cdp/volha_ivanova_task1/Runner.java

```diff
@@ -0,0 +1,35 @@
+package com.epam.cdp.volha_ivanova_task1;
+
+public class Runner {                       //Volha Ivanova. В массиве целых чисел определить сумму элементов, состоящих на чётных позициях
+
+	int [] arr1;
```

Comment:
According to the name of variables : don't use this type of names. Name of variable should be meaningful. Now I dont understand for what purpose the var is.
Please update all variables with meaningful names

Additionally I see that you don't use the var. You create additional one in main method

In the future if you will use fields in class don't forget about access modifiers. It should be private in ordinary case